packages/webpack/src/plugin.js

```diff
@@ -206,13 +238,19 @@ class LavaMoatPlugin {
           diag.rawDebug(4, { knownPaths })
           PROGRESS.report('pathsCollected')
 
+          diag.rawDebug(2, 'writing policy')
+          // use the generated policy to save the user one additional pass
+          // getting the policy also writes all files where necessary
+          options.policy = policyGenerator.getPolicy()
```

Comment:
does it matter if we're mutating `options`

Reply:
not sure what you mean

Reply:
ok, I noticed I probably don't need to write it back to options anyway